src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +50,208 @@ export const KafkaPage: FunctionComponent = () => {
             <CardHeader>
               <CardHeaderMain>
                 <CardTitle>
-                  <Title headingLevel="h3">
-                    {t("kafkaoverview:purchaseCardTitle")}
-                  </Title>
+                  <Title headingLevel="h3">{t("purchaseCardTitle")}</Title>
```

Comment:
I see there's an `<h1>` element but not an `<h2>`. All of the card titles should be updated to use `headingLevel="h3"`

Reply:
To clarify this comment further, the first header on every page should always be h1. The next header should be h2 and not higher. In theory, there should only ever be one h1 on a page, which is the main title. So then the next header can be either h2 for a new section, or h3 if creating a sub-section. 

Also note, I only posted comments for the titles where the heading level should be updated.